code/modules/antagonists/nukeop/clownop.dm

```diff
@@ -41,6 +42,7 @@
 	antagpanel_category = "ClownOp"
 	nukeop_outfit = /datum/outfit/syndicate/clownop/leader
 	challengeitem = /obj/item/nuclear_challenge/clownops
+	suicide_cry = "HAPPY BIRTHDAY!!"
```

Comment:
inherited from parent

Reply:
the clownop leader's parent datum is `/datum/antagonist/nukeop/leader`, instead of `/datum/antagonist/nukeop/clownop`

Reply:
oh god damn it